packages/salesforce-adapter/src/filters/flows_filter.ts

```diff
@@ -47,62 +47,103 @@ import { SalesforceRecord } from '../client/types'
 const { isDefined } = lowerdashValues
 
 const log = logger(module)
+const { DefaultMap } = collections.map
 const { toArrayAsync } = collections.asynciterable
 
 const DEFAULT_CHUNK_SIZE = 500
 
-const fixFilePropertiesName = (props: FileProperties, activeVersions: Map<string, string>): FileProperties => ({
-  ...props,
-  fullName: activeVersions.get(`${props.fullName}`) ?? `${props.fullName}`,
-})
+type FlowVersionRecord = SalesforceRecord & {
+  DefinitionId: string
+  VersionNumber: number
+  Status: string
+  CreatedDate: string
+  CreatedBy: {
+    Name: string
+  }
+  LastModifiedDate: string
+  LastModifiedBy: {
+    Name: string
+  }
+}
+const isFlowVersionRecord = (record: SalesforceRecord): record is FlowVersionRecord =>
+  _.isString(record.DefinitionId) &&
+  _.isString(record.Status) &&
+  _.isNumber(record.VersionNumber) &&
+  _.isString(record.CreatedDate) &&
+  _.isPlainObject(record.CreatedBy) &&
+  _.isString(record.CreatedBy.Name) &&
+  _.isString(record.LastModifiedDate) &&
+  _.isPlainObject(record.LastModifiedBy) &&
+  _.isString(record.LastModifiedBy.Name)
```

Comment:
```suggestion
  _.isString(record, ['LastModifiedBy', 'Name'])
```
nit: Consider the following

Reply:
This isn't a thing, maybe you meant `_.isString(_.get(record, ['LastModifiedBy', 'Name'])` which I actually like less (generally I think `_.get` is a cop out).